src/ConstructiveSolidGeometry/LinePrimitives/Line.jl

```diff
@@ -4,3 +4,9 @@ struct Line{T} <: AbstractLinePrimitive{T}
 end
 
 distance(pt::CartesianPoint, l::Line) = norm((pt - l.origin) × l.direction) / norm(l.direction)
+
+function _transform_into_object_coordinate_system(l::Line{T}, p::AbstractPrimitive) where {T}
+    origin = _transform_into_object_coordinate_system(l.origin, p) 
+    direction = _transform_into_object_coordinate_system(CartesianPoint{T}(l.direction), p)
+    Line{T}( origin, direction )  
```

Comment:
Shouldn't `direction` be a `CartesianVector` rather than a `CartesianPoint`?
Or is that converted when constructing the `Line`?

Reply:
That's automatically converted as both are basically SVector{3,T}.
But I will change it for readability.